<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TedarikciDuzenleme</class>
 <widget class="QWidget" name="TedarikciDuzenleme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>356</width>
     <height>305</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblTedarikciAdi">
        <property name="text">
         <string>Tedarikçi Adı</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="leTedarikciAdi"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblTedarikciAdresi">
        <property name="text">
         <string>Tedarikçi Adresi</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPlainTextEdit" name="pteTedarikciAdresi"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblTedarikciTelefonu">
        <property name="text">
         <string>Tedariçi Telefonu</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="leTedarikciTelefon">
        <property name="inputMask">
         <string>\o(D99)999 99 99</string>
        </property>
        <property name="maxLength">
         <number>15</number>
        </property>
        <property name="cursorPosition">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="lblTedarikciYetkiliKisi">
        <property name="text">
         <string>Tedarikçi Yetkili Kişi</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="leTeadrikciYetkiliKisi"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnKaydet">
        <property name="text">
         <string>Kaydet</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnIptal">
        <property name="text">
         <string>İptal</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
